core/modules/user/user.module

```diff
@@ -3095,6 +3094,17 @@ function user_form_field_ui_field_edit_form_submit($form, &$form_state) {
   }
 }
 
+/** 
+ * Menu callback to create a new account. 
+ * 
+ * @see user_register_form()
+ */
```

Comment:
Could use an `@since 1.23 Replaces user_register_form() as menu callback` or something such. Same for other function.

Reply:
This has been addressed in the latest push